scripts/monitoring/cron-send-pcp-sampled-metrics.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+'''
+  Sample pcp for cpu statistics over an interval
+
+  Example:
+  ./pcp_sample.py -m 'kernel.all.cpu.idle,kernel.all.cpu.nice,kernel.all.cpu.steal,kernel.all.cpu.sys,\
+kernel.all.cpu.user,kernel.all.cpu.wait.total' -i 2
+'''
+
+import sys
+import argparse
+from openshift_tools.monitoring import pminfo
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+
+def parse_args():
+    '''Parse the arguments for this script'''
+    parser = argparse.ArgumentParser(description="Tool to sample pcp metrics")
+    parser.add_argument('-m', '--metrics', action="store",
+                        help="Comma delimited string of metrics to send to zabbix")
```

Comment:
I'd prefer this to be able to take multiple -m arguments.

This looks nicer and makes more sense to me than a comma separated list.

Otherwise the comma separated list becomes a wall of text to my eyes.

This is how ansible works with -e, docker works with -v, -p, etc. I think that's a more established pattern.

```
/usr/local/bin/pcp_sample.py -m kernel.all.cpu.idle -m kernel.all.cpu.nice -m kernel.all.cpu.steal -m kernel.all.cpu.sys -m kernel.all.cpu.user -m kernel.all.cpu.wait.total
```